rustSlint/Dockerfile.debug

```diff
@@ -34,7 +33,8 @@ ARG GPU=
 ##
 # Deploy Step
 ##
-FROM commontorizon/slint-base-${IMAGE_ARCH}${GPU}:${BASE_VERSION} AS debug
+FROM --platform=linux/${IMAGE_ARCH} \
+    torizon/wayland-base${GPU}:${BASE_VERSION} AS debug
```

Comment:
Known issue here is that for this image to work, we also need to install libslint_cpp.so*. That was previously in the base image.